pkgs/shells/powershell/default.nix

```diff
@@ -47,6 +47,8 @@ stdenv.mkDerivation rec {
 
     ls $pslibs
   '' + lib.optionalString (!stdenv.isDarwin && !stdenv.isAarch64) ''
+    mkdir -p $out/lib
+    ln -s ${lttng-ust}/lib/liblttng-ust.so.1 $out/lib/liblttng-ust.so.0
```

Comment:
I feel like the suffix is there for abi reasons, not sure if I'm comfortable feeding the binary lies.

Reply:
I think this PR can be merged for temporary needs. We can create a new PR after the Powershell upstream fixed the problem.